transport_nantes/topicblog/models.py

```diff
@@ -960,9 +972,21 @@ class TopicBlogPressSendRecord(models.Model):
 
     """Represent the fact that we sent an press.
     """
+
+    class StatusChoices(models.TextChoices):
+        # This is the default state
+        PENDING = 'PENDING', "Pending"
+        # Toggled to this state when Amazon SES sends the email
+        SENT = 'SENT', "Sent"
+        # Toggled to this state if Amazon SES refuses to send the email
+        FAILED = 'FAILED', "Failed"
+
     slug = models.SlugField(max_length=90, allow_unicode=True, blank=True)
     mailinglist = models.ForeignKey(MailingList, on_delete=models.PROTECT)
     recipient = models.ForeignKey(User, on_delete=models.PROTECT)
+    status = models.CharField(
+        max_length=10, choices=StatusChoices.choices,
```

Comment:
PostgreSQL doesn't care about `max_length`, but django does.  It's generally wise to give yourself extra space to avoid having to fiddle with the schema definition because you add an enum element.